Add notes on configuring LDAP STS with Microsoft Active Directory (#8260)
Minio-io/minio: docs/sts/ldap.md

**MINIO_IDENTITY_LDAP_GROUP_SEARCH_BASE_DN** environment variables
support substitution of the *username* and *usernamedn* variables
only.

## Notes on configuring with Microsoft Active Directory (AD)

> The LDAP STS API also works with Microsoft AD and can be configured in a manner

Reviewer: ```suggestion
The LDAP STS API also works with Microsoft AD and can be configured
```

> similar to the above. The following are some notes on determining the values of

Reviewer: ```suggestion
as above. The following are some notes on determining the values of
```
